app/models/appointment.rb

```diff
@@ -0,0 +1,19 @@
+class Appointment < ApplicationRecord
+  belongs_to :user
```

Comment:
I wonder if it would be more helpful to refer to `guider` here instead? It seems we have a bunch of places now where we could provide these useful cues to disambiguate.